src/Infrastructure/Cache/File/README.md

```
@@ -0,0 +1,8 @@
+# File Cache Adapter
+
+File system-based caching implementation:
+- Simple file-based storage
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)